pipeline/tracer.go

```diff
@@ -17,6 +17,8 @@ package pipeline
 import (
 	"strconv"
 	"time"
+
+	"go.woodpecker-ci.org/woodpecker/v2/server/model"
```

Comment:
pipeline should have it's own enums if i remember correctly

in any case we dont want a circular dependency

Reply:
Makes sense, but how should this case be handled? Copy the status consts from the servers model package to the pipeline package?

Reply:
make a copy and put it into pipeline/const.go migt be the best option atm.
later we could use that type in server code instead

Reply:
Done